mpv-plugin/SimpleHistory/SimpleHistory-2023.09.25.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+USE_MPV="rdepend"
+MPV_REQ_USE="lua"
+inherit mpv-plugin
+
+DESCRIPTION="Continue watching last played, manage and play from history"
+HOMEPAGE="https://github.com/Eisa01/mpv-scripts#simpleundo"
+SRC_URI="https://github.com/Eisa01/mpv-scripts/archive/refs/tags/25-09-2023.tar.gz -> ${P}.tar.gz"
```

Comment:
> 25-09-2023

Put this in a variable? Ideally (though not necessarily) automatically worked out from `${PV}`.

Reply:
Looks like I cannot extract to `${PV}` because the year must be first?

Reply:
You can extract individual component thru [parameter expansion](https://github.com/dylanaraps/pure-sh-bible#parameter-expansion):

```bash
V_Y=${PV%%.*}
V_M=${${PV#*.}%.*}
V_D=${PV##.*}
```

And then reshuffle it into a single variable which will be updated automatically from `${PV}`:

```bash
MY_PV="${V_D}-${V_M}-${V_Y}"
```

Reply:
Wow, thanks, implemented.